docs/en/log/flight_log_analysis.md

```diff
@@ -41,6 +41,23 @@ Key features:
 
 See [Log Analysis using Flight Review](../log/flight_review.md) for an introduction.
 
+### Foxglove
+
+[Foxglove](https://foxglove.dev/) is a purpose-built robotics observation platform that works natively with ULog.
+It allows you to replay your flights and scrub through the timeline to find data of interest.
```

Comment:
@msadowski Thanks for this. I've moved the section higher up the list, as a first step to ordering this alphabetically (the fairest approach for lists unless there is a clear defacto standard for a particular options).

The only question is "what does it mean to scrub through a timeline"?

Reply:
Thanks for the review Hamish!
 
We applied this terminology from the recording industry ([example](https://manual.audacityteam.org/man/scrubbing_and_seeking.html)). I changed it to "seek through" as I think it's more intuitive and widely known